.env

```diff
@@ -11,4 +11,5 @@ BUILD_NUMBER=STAGING::_build_number_
 BUILD_TYPE=development
 OPEN_SEA_API=https://rinkeby-api.opensea.io/api/v1
 OPEN_SEA_API_KEY=
+BADGES_CONTRACT_ADDRESS=0xe32e4DbA36675f1F2Fff3a02b399b0CF1D80EEe6
```

Comment:
@jsidorenko @alazarevski @poocart  is it a problem that the CONTRACT_ADDRESS is committed in the repo ?

Reply:
Nope, it's public information